Userland/Libraries/LibWeb/CSS/StyleValues/RotationStyleValue.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024, Steffen T. Larssen <dudedbz@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/String.h>
+#include <LibWeb/CSS/StyleValues/CSSMathValue.h>
+#include <LibWeb/CSS/StyleValues/NumberStyleValue.h>
+
+#include "RotationStyleValue.h"
+
+namespace Web::CSS {
+
+// https://www.w3.org/TR/2021/WD-css-transforms-2-20211109/#individual-transform-serialization
+String RotationStyleValue::to_string() const
+{
+    auto resolve_to_number = [](ValueComparingNonnullRefPtr<CSSStyleValue const> const& value) -> double {
+        if (value->is_number())
+            return value->as_number().number();
+        if (value->is_math() && value->as_math().resolves_to_number())
+            return value->as_math().resolve_number().value();
+
+        VERIFY_NOT_REACHED();
+    };
```

Comment:
If the calculated number isn't resolvable then this will crash. My suggestion would be:
- Make this lambda return `Optional<double>`
- Then it can `return value->as_math().resolve_number();` directly
- The `x_value == 0` and similar comparisons will still work without changes. (You can compare `Optional<T>` to `T` and it'll return true if it has a value and that value is equal)
- (see below)